server/api/api/endpoints/artifacts_v2.py

```diff
@@ -199,6 +199,7 @@ async def get_artifact(
     tag: str = None,
     iter: int = None,
     object_uid: str = Query(None, alias="object-uid"),
+    uid: str = Query(None),
```

Comment:
I think it should work:
```suggestion
    uid: str = Query(None, alias="object-uid"),
```

Reply:
I wouldn't want to remove `object_uid` as it will remove BC.